lib/modules/hosts/iloopit.js

```diff
@@ -3,32 +3,26 @@ addLibrary('mediaHosts', 'iloopit', {
 	domains: ['iloopit.net'],
 	logo: '//iloopit.net/favicon.ico',
 	detect: href => (
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/tube\/(\d+)/i).exec(href) ||
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/[\-a-z\ \/%20]+?\/\?type=looplayer&loopid=(\d+)/i).exec(href) ||
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/(\d+)\/[a-z\-\/]+((\?type=looplayer)|(\?type=embed))?/i).exec(href)
+		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/.+?\/\?type=looplayer&loopid=(\d+)/i).exec(href) ||
+		(/^https?:\/\/(?:\w+\.)?iloopit\.net(?:\/tube\/)?\/(\d+)\/.+?\/(?:(?:\?type=looplayer)|(?:\?type=embed))?/i).exec(href)
```

Comment:
`(?:(?:\?type=looplayer)|(?:\?type=embed))?` is redundant and should be removed (it's optional, contains no capture groups, and is not followed by `$` or another group, so it can't affect the match).